cmd/agent-js-worker/README.md

```diff
@@ -40,9 +40,29 @@ const agent = await new Agent.Framework({
     "auto-accept": true,
     "outbound-transport": ["ws", "http"],
     "transport-return-route": "all",
-    "indexedDB-namespace": "demo",
+    "log-level": "debug",
     "storageType": "indexedDB",
-    "log-level": "debug"
+    "indexedDB-namespace": "demo",
+    "edvServerURL": "",
+    "edvVaultID": "",
+    "edvCapability": "",
+    "blocDomain": "",
+    "trustbloc-resolver": "",
+    "authzKeyStoreURL": "",
+    "opsKeyStoreURL": "",
+    "edvOpsKIDURL": "",
+    "edvHMACKIDURL": "",
+    "kmsType": "",
+    "userConfig": {
+        "accessToken": "",
```

Comment:
This will likely be removed very soon.

Reply:
Ok, I am removing `accessToken` and `opsKMSCapability`